### PR TITLE
DLP: Added sample for update trigger

### PR DESCRIPTION
**Pull request by jainamtrivedi-crest**
- Added sample for [Update Trigger](https://cloud.google.com/dlp/docs/creating-job-triggers#update_an_existing_job_trigger) and unit test cases.

### CODE REVIEW

**Review by annadata — APPROVED** [review]
Pl address minor comments.

File: dlp/api/Snippets/TriggersUpdate.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                            infoTypes ?? new InfoType[]
+                            {
+                                new InfoType { Name = "US_INDIVIDUAL_TAXPAYER_IDENTIFICATION_NUMBER" }
+                            }
+                        },
+                        MinLikelihood = minLikelihood
+                    }
+                }
+            },
+            UpdateMask = new FieldMask
```
[the remark this answers is not in the document]
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/TriggersUpdate.cs:59]: Done. Thanks

**Comment by jainamtrivedi-crest** [comment]
CI is getting failed because the issue where buckets were not getting deleted properly in create and update stored infoType test cases. Raised a fix PR: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2320

File: dlp/api/Snippets.Tests/TriggersUpdateTests.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        {
+            // Create a trigger first.
+            var infoTypes = new InfoType[] { new InfoType { Name = "PERSON_NAME" } };
+            var triggerId = $"my-csharp-test-trigger-{Guid.NewGuid()}";
+            var fullTriggerId = $"projects/{_fixture.ProjectId}/jobTriggers/{triggerId}";
+            var displayName = $"My trigger display name {Guid.NewGuid()}";
+            var description = $"My trigger description {Guid.NewGuid()}";
+
+            using var randomBucketFixture = new RandomBucketFixture();
+            using var bucketCollector = new BucketCollector(randomBucketFixture.BucketName);
```
Comment (amanda-tarafa) [inline dlp/api/Snippets.Tests/TriggersUpdateTests.cs:39]: Consider using these for #2300 as well.

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**